src/plugins/ui_core_views/cell_evaluation/evaluator.ts

```diff
@@ -507,7 +530,7 @@ class PositionBitsEncoder {
   private decodeSheet(sheetKey: PositionId): UID {
     const sheetId = this.inverseSheetMapping.get(sheetKey);
     if (sheetId === undefined) {
-      throw new Error("Sheet id not found");
+      throw { value: ERR_GENERIC, message: "Sheet id not found" };
```

Comment:
This is a real programming error. It should remain an `Error`

Reply:
yes, my bad